### ISSUE TITLE
Who is maintaining this repo and can we delegate admin for maintenance?

### ISSUE THREAD

**Issue by webgismd** [BCDevOps/certbot#17]
@cvarjao @jujaga

**Comment by webgismd** [comment]
@juhewitt -- is there a central project around automation for SSL Certificates? 

Thank you!!

**Comment by juhewitt** [comment]
@NickCorcoran

**Comment by NickCorcoran** [comment]
@webgismd @slhurley Sorry for the super slow response on this.  I must have missed the tag.  
There is no central project around automating SSL cert provisioning/renewal at this time.  I have tried to bring up a few times, but there seems to be some hesitation, specifically around billing.  I will try to broach the subject again though.  
I have documented out process re: obtaining certs here: https://developer.gov.bc.ca/BC-Government-OpenShift-DevOps-Security-Considerations#tls-certificates

As far as use of the repo, feel free to fork it and then issue a PR back when complete.

I know there is also interest in using the cert manager operator for OpenShift (https://github.com/openshift/cert-manager-operator).  I have plans to look into that at some point...
